Sources/Orbit/Components/Icon.swift

```diff
@@ -43,8 +48,13 @@ public struct Icon: View {
                     .accessibility(hidden: true)
             case .countryFlag(let countryCode):
                 CountryFlag(countryCode, size: size)
-            case .sfSymbol(let systemName):
+            case .sfSymbol(let systemName, let color?):
+                Image(systemName: systemName)
+                    .foregroundColor(color)
+                    .font(.system(size: size.value * Self.sfSymbolToOrbitSizeRatio * sizeCategory.ratio))
```

Comment:
No accessibility for SF Symbols?

Reply:
The SF Symbols already support accessibility. (can be overridden by caller if the native version is not appropriate)